pkgs/top-level/all-packages.nix

```
@@ -7551,6 +7551,10 @@ in
 
   yeshup = callPackage ../tools/system/yeshup { };
 
+  ytop = callPackage ../tools/system/ytop {
+    inherit (darwin) IOKit;
```

nitpick
```suggestion
    inherit (darwin.apple_sdk.frameworks) IOKit;
```
besides that, LGTM

---

Fixed.